scripts/ci/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e # exit on error
+set -x # echo commands
+
+# Workaround for Travis-CI bug #2: https://github.com/travis-ci/travis-ci/issues/7773
+if ! [ "$TRAVIS_OS_NAME" = "linux" ]; then
+    EXTRA_OPTS="--no-test"
+else
+
+echo "Building conda package."
+conda build -c defaults -c conda-forge ${EXTRA_OPTS:-} ./conda
+
+# If tagged, upload package to main channel, otherwise if not a PR, upload to dev
+if [ -n "$TRAVIS_TAG" ]; then
+    LABEL_OPT=""
+elif ! [ -n "$TRAVIS_PULL_REQUEST" ]; then
```

Comment:
We should take this opportunity to stop pushing dev packages at all, i.e., the upload only happens for the tag branch

Reply:
done in https://github.com/ContinuumIO/intake/pull/212/commits/9b151d3da9b7f88b3a7d140b7041ff029a55a109 (also fixed logic and syntax errors)